docs/faq.md

```diff
@@ -0,0 +1,53 @@
+---
+id: faq
+title: Frequently Asked Questions
+sidebar_label: FAQ - Questions
+---
+
+## What is the recommended single-spa setup?
+We recommend a setup that uses ES modules + [import maps](https://github.com/WICG/import-maps) + [dynamic import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Dynamic_Imports) (or [SystemJS](https://github.com/systemjs/systemjs) to polyfill these if you need better browser support).  This setup has several advantages:
+1. Common libraries are easy to manage, and are only downloaded once. You can also preload these for a small speed boost as well using the standard preload spec.
+2. Sharing code / functions / variables is as easy as an import statement, just like in a monothlithic setup
+3. Lazy loading applications is easy, which enables you to speed up initial load times
+4. Each application (AKA microservice, AKA ES module) can be independently developed and deployed. Teams are enabled to work at their own speed, experiment (within reason as defined by the organization), deploy, and QA on thier own schedules.
+5. Working on one area of an app doesn’t require you to download the entire frontend codebase — you just run your microservice locally and the other ones are automatically pulled from what’s on your dev/stage/prod environment
+
+## Is there a performance hit to doing microservices this way?
+When setup in the [recommended way](#what-is-the-recommended-single-spa-setup), your code performance and bundle size will be the
```

Comment:
Should this sentence be left unfinished? If you'll be adding another PR before this gets built out to the site, that should be fine.

Reply:
oh haha probably just missed it.